### PR TITLE
Fix indentation

### PR DESCRIPTION
**Pull request by JeffreyCA**

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[1b78b84](https://github.com/JeffreyCA/docs.microsoft.com-nuget/pull/1377/commits/1b78b8457b39450baafab5c3b6dae894728bc3a2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/nuget-config-file.md](https://github.com/JeffreyCA/docs.microsoft.com-nuget/blob/patch-2/docs/reference/nuget-config-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-config-file?branch=pr-en-us-1377) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C2%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904022045377513-1377%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=lbIERziQhfjcIaFkvbRPMw%2BLspFzm%2BoW46F2SBY1GDc%3D&st=2019-04-02T20%3A41%3A49Z&se=2019-05-03T20%3A46%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/nuget-config-file.md [hunk 1]
```diff
@@ -347,12 +347,12 @@ Below is an example `nuget.config` file that illustrates a number of settings:
     -->
     <trustedSigners>
         <author name="microsoft">
-		    <certificate fingerprint="3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE" hashAlgorithm="SHA256" allowUntrustedRoot="false" />
```
Comment (loic-sharma) [inline docs/reference/nuget-config-file.md]: This line still has a tab. See: https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#example-config-file

**Comment by loic-sharma** [comment]
It looks like there's still one last tab left, please address it and I'll merge your changes. Thanks for fixing this!

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[c021c32](https://github.com/JeffreyCA/docs.microsoft.com-nuget/pull/1377/commits/c021c32e66f8d3048de420d071635df57622579e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/nuget-config-file.md](https://github.com/JeffreyCA/docs.microsoft.com-nuget/blob/patch-2/docs/reference/nuget-config-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-config-file?branch=pr-en-us-1377) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C3%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904031537536623-1377%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=8VfmGMsUBfjwNE9F8FJZMX7MhfyB9GkWF6rFzuujNns%3D&st=2019-04-03T15%3A34%3A07Z&se=2019-05-04T15%3A39%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**